### PR TITLE
Create 7flash.json

### PR DESCRIPTION
**Pull request by 7flash**

### CODE REVIEW

File: public/data/7flash.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "avatar": "https://avatars.githubusercontent.com/u/4569866?v=4",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/7flash",
+      "icon": "github"
+    }
+  ],
+  "milestones": [
+        "title": "Joined Github",
```
Comment (Panquesito7) [inline public/data/7flash.json:14]: I recommend capitalizing the `h` here. What do you think?
```suggestion
        "title": "Joined GitHub",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: public/data/7flash.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Berlian Gur",
+  "type": "personal",
+  "bio": "Software Engineer",
+  "avatar": "https://avatars.githubusercontent.com/u/4569866?v=4",
```
Comment (Panquesito7) [inline public/data/7flash.json:5]: It is best to use this link as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/7flash.png",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: public/data/7flash.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  "avatar": "https://github.com/7flash.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/7flash",
+      "icon": "github"
+    }
+  ],
+  "milestones": [
+        "title": "Joined GitHub",
```
Comment (eddiejaoude) [inline public/data/7flash.json:14]: This data needs to be wrapped in an object `{}`

For example

```json
      "milestones": [{
         "title": "Joined GitHub",
         "date": "December 2013"
       }]
```

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**